CHANGES

```diff
@@ -4,6 +4,7 @@ XXX:
   - AutoDoc can now be instructed to invoke GAPDoc in such a way that links
     in the generated documentation to the GAP reference manual use relative
     paths. To do this, use TODO
```

Comment:
The above incomplete CHANGES entry should be finished. Perhaps just remove the last incomplete sentence (the stuff before it should be enough, the user can find details in the documentation now).